lib/cuckoo/common/abstracts.py

```diff
@@ -1041,6 +1041,18 @@ def check_url(self, pattern, regex=False, all=False):
                                  regex=regex,
                                  all=all)
 
+    def check_suricata_alerts(self, pattern):
+        """Check for pattern in Suricata alert signature
+        @param pattern: string or expression to check for.
+        @return: True/False
+        """
+        res = False
+        for alert in self._caller.results.get("suricata", {}).get("alerts", []):
+            if re.findall(pattern, alert.get("signature", ""), re.I):
+                res = True
```

Comment:
Please replace this logic by a `return True` here and a `return False` at the end of the function and then I'll merge it!